qcengine/procedures/base.py

```diff
@@ -0,0 +1,53 @@
+"""
+Imports the various procedure backends
+"""
+
+from typing import List, Set
+
+from .geometric import GeometricProcedure
```

Comment:
Didn't we add a second procedure and service in Fractal? Is there a reason that could not be added here? If not, will Fractal's additional procedure derive from the abstract base model?

Reply:
Procedures will be here in Engine, all Services will stay in Fractal as they need Fractal-level protocols. Engine is also pure input/output QCSchema, Fractal records are Schema + stuff archival items.